packages/turf-clean-coords/index.ts

```diff
@@ -39,7 +43,7 @@ function cleanCoords(geojson, options) {
         });
         break;
     case 'MultiPolygon':
-        getCoords(geojson).forEach(function (polygons) {
+        getCoords(geojson).forEach(function (polygons: any) {
```

Comment:
`getCoords` is a bit wacky now, not outputting the correct type, `any` was added as a "hack" solution.